crates/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -403,7 +403,14 @@ fn execute_move_publish<E: fmt::Debug, S: StorageView<E>, Mode: ExecutionMode>(
         })
         .collect::<Vec<_>>();
 
-    let package_id = context.new_package(modules)?;
+    // new_package also initializes type origin table in the package object
+
+    // TODO: need to pass actual dependencies from publish/upgrade command line (converted to a
+    // slice of MovePackage's; panic for now as it will fail down the road anyway and this way it's
+    // easier to spot.
+    panic!("Package dependencies not yet available");
```

Comment:
Because of this panic, we'll probably want to wait to merge @rvantonder's PR into this one, and then land them both together into `main`.

Reply:
@awelc feel free to just remove this TODO/`panic` at this stage, since we know we'll be merging into this PR before landing in `main`. `dep_ids` is available in this function from my  PR ([relevant line](https://github.com/MystenLabs/sui/pull/9174/files#diff-a7c252ee069aebe25afd331dc951b6190d92bb90442efa5dd37dec2190fe6567R383))